db/patients.js

```diff
@@ -16,11 +16,12 @@ module.exports.getPatients = async () => {
 
 const insertPatient = async (firstNames, lastName, studentNumber) => {
   try {
-    return await knex(module.exports.patientTable).insert({
+    const patientsArray = await knex(module.exports.patientTable).insert({
       firstNames,
       lastName,
       INE: studentNumber,
-    })
+    }).returning('*')
+    return patientsArray[0]
```

Comment:
il me semble que `first()` marche
```suggestion
    return patientsArray.first()
```

Reply:
`first` ca existe pas en js, ca existe dans `knex` par contre. Mais ca m'a dit `Cannot chain .first() on "insert" query` donc j'ai laissé tombé, c'est sans doute pas pour ca que ca sert.